cassandra/connection.py

```diff
@@ -721,7 +723,8 @@ def _handle_options_response(self, options_response):
     @defunct_on_error
     def _send_startup_message(self, compression=None, no_compact=False):
         log.debug("Sending StartupMessage on %s", self)
-        opts = {}
+        opts = {'DRIVER_NAME': DRIVER_NAME,
+                'DRIVER_VERSION': DRIVER_VERSION}
```

Comment:
so that will just pass with older versions as well? Probably simply ignored?

Reply:
Yep, seems to just be ignored.